buildlib/az-network-corrupter.sh

```diff
@@ -3,17 +3,19 @@
 echo "Running $0 $*..."
 eval "$*"
 initial_delay=${initial_delay:=10}
+interface=${interface:=bond0}
 cycles=${cycles:=1000}
 downtime=${downtime:=5}
 uptime=${uptime:=20}
 reset=${reset:="no"}
 
 
-manager_script=/hpc/noarch/git_projects/swx_infrastructure/clusters/bin/manage_host_ports.sh
+manager_script_dir=/hpc/noarch/git_projects/hpc-mtt-conf/scripts
+manager_script=${manager_script_dir}/switch_port_on_off.py
 
 if [ "x$reset" = "xyes" ]; then
-    echo "Resetting interface on $(hostname)..."
-    ${manager_script} "$(hostname)" "bond-up"
+    echo "Resetting interface on $(hostname) interface ..."
+    python ${manager_script} -d ${manager_script_dir}/hosts --host $(hostname) -a on -i ${interface}
```

Comment:
why need python? IMO the script should be executable

Reply:
switch port up/down on python , take from mtt , for avoid double support for a same scripts

Reply:
can't just run
`${manager_script} -d ${manager_script_dir}/hosts --host $(hostname) -a on -i ${interface}`
why need "python" in the command line?

Reply:
this is file no executable

Reply:
can you make it executable?

Reply:
done